src/typeahead/typeahead.spec.ts

```diff
@@ -473,7 +473,7 @@ describe('ngb-typeahead', () => {
            const fixture = createTestComponent(`<input type="text" [(ngModel)]="model" 
                 [ngbTypeahead]="findAnywhere" 
                 [inputFormatter]="uppercaseFormatter" 
-                [showHint]="true"/>`);
+                [showHint]="true" [autoSelectItem]="true" />`);
```

Comment:
We should _not_ modify existing tests. `autoSelectItem` should be default. Revert.